### DISCUSSION TITLE
Admin permission

### DISCUSSION THREAD

**Discussion by barryzhounb** [strimzi/strimzi-kafka-operator#4659] [General]
When I use Kafka Admin to fetch topic list, do I need any permissions?

**Comment by scholzj** [comment]
Yes. It will probably need some Describe ACL.

**Reply by barryzhounb** [reply]
Is it possible for me that an User have a permission with only Describe specific topics rather than all topics?

**Reply by scholzj** [reply]
I'm not an expert on the Kafka ACLs. But you should be able to set the Describe ACL on a topic level. So it should be possible.

**Reply by barryzhounb** [reply]
thanks
